python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -154,18 +153,26 @@ def getDestination(self, butler, info, filename):
         raw : `str`
             Destination filename.
         """
-        # Arbitrarily set ccdnum and calib_hdu to 1 to make the mapper template happy
-        info["ccdnum"] = 1
-        info["calib_hdu"] = 1
         calibType = self.getCalibType(filename)
-        if "flat" in calibType.lower():
-            raw = butler.get("cpFlat_filename", info)[0]
-        elif ("bias" or "zero") in calibType.lower():
-            raw = butler.get("cpBias_filename", info)[0]
-        elif ("illumcor") in calibType.lower():
-            raw = butler.get("cpIllumcor_filename", info)[0]
-        else:
-            assert False, "Invalid calibType '{:s}'".format(calibType)
+        md = readMetadata(filename, self.config.hdu)
+        if "PROCTYPE" not in md:
+            raise RuntimeError(f"Unable to find the calib header keyword PROCTYPE \
+                               in {filename}, hdu {self.config.hdu}")
+        proctype = md["PROCTYPE"].strip()
+        if "MasterCal" in proctype:  # CP calibration product case
```

Comment:
Please expand this acronym, since we have our own `cp_pipe`: "DECam Community Pipeline calibration product"